### PR TITLE
Update resolving-a-merge-conflict-on-github.md

### PR DESCRIPTION
**Pull request by exp-horibe**
How to resolve conflicts involving non-text format files.

### Why:

Closes [issue link]

### Check off the following:

- [ ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@exp-horibe 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by aleksan1 — COMMENTED** [review]
Nice job

**Review by aleksan1 — COMMENTED** [review]
@exp-horibe #

> 

> .github/workflows/auto-label-prs.yml

**Comment by aleksan1** [comment]
> @exp-horibe #
> 
> > 
> 
> > .github/workflows/auto-label-prs.yml

**Review by aleksan1 — COMMENTED** [review]
https://github.com/github/docs/pull/16277#pullrequestreview-915011753

File: content/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github.md [hunk 1]
```diff
@@ -59,6 +59,11 @@ If a site administrator disables the merge conflict editor for pull requests bet
    Click **Create branch and update my pull request** or **I understand, continue updating _BRANCH_**. The button text corresponds to the action you are performing.
 1. To merge your pull request, click **Merge pull request**. For more information about other pull request merge options, see "[Merging a pull request](/articles/merging-a-pull-request/)."
 
+ If the conflict is in binary data such as an image or sound file, this method of resolving conflicts cannot be used. 
+ In order to resolve conflicts involving binary files, immidiately make a back up, delete and re-clone the project, then re-introduce non-conflicting files.
```
Comment (hubwriter) [inline content/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github.md:63]: I'm not sure that advising the user to add non-conflicting files is particularly useful in this article. This also assumes the user is working from a clone, which may not be the case.

File: content/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github.md [hunk 2]
```diff
@@ -59,6 +59,11 @@ If a site administrator disables the merge conflict editor for pull requests bet
...
+ 
+ For the conflicting files, examine the latest data in the freshly cloned project and decide if you can append your edits in a DCC program or whether to overwrite the changes entirely in a fresh pull request.
```
Comment (hubwriter) [inline content/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github.md:65]: I don't know what a DCC program is. If referring to this you will need to explain what it is, and possibly link to more information elsewhere.

**Comment by Truthofman13513** [comment]
> How to resolve conflicts involving non-text format files.
> 
> ### Why:
> Closes [issue link]
> 
> ### What's being changed:
> ### Check off the following:
> * [ ]  I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
> * [ ]  For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).
> 
> ### Writer impact (This section is for GitHub staff members only):
> * [ ]  This pull request impacts the contribution experience
>   
>   * [ ]  I have added the 'writer impact' label
>   * [ ]  I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

thanks

**Comment by cmwilson21** [comment]
We are going to go ahead and close this issue. If you are able to provide the requested information above and would like to work on it more, we can reopen it again. ✨ 

Thanks for your passion for contributing to our docs! 💛

**Pull request closed without merging**